src/main/java/com/icthh/xm/tmf/ms/communication/service/MobileAppMessagePayloadCustomizer.java

```diff
@@ -0,0 +1,22 @@
+package com.icthh.xm.tmf.ms.communication.service;
+
+import com.icthh.xm.commons.lep.LogicExtensionPoint;
+import com.icthh.xm.commons.lep.spring.LepService;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+/**
+ * Allows to customize mobile application payload being sent to the mobile device
+ */
+@Component
```

Comment:
seems @Component is not needed as you have @LepService

Reply:
removed